src/mappings/entities/identities/mapClaim.ts

```diff
@@ -57,7 +57,7 @@ const getId = (
   if (scope) {
     // Not applicable in case of CustomerDueDiligence, InvestorUniquenessV2Claim, NoData claim types
     idAttributes.push(scope.type);
-    idAttributes.push(scope.value);
+    idAttributes.push(scope.assetId ?? scope.value);
```

Comment:
Since we have added this, resync will make sure that assetID is always used. If we update the existing asset scope types to use assetId  irrespective of spec version, we won't need to handle chain upgrade